parser-typechecker/src/Unison/Codebase/Editor/Propagate.hs

```diff
@@ -146,8 +169,26 @@ propagate patch b = case validatePatch patch of
         (Set.fromList
           [ r | Referent.Ref r <- Set.toList $ Branch.deepReferents b ]
         )
-    initialDirty <-
-      R.dom <$> computeFrontier (eval . GetDependents) patch names0
+      ns = traceShowId $ Branch.toNames0 b
```

Comment:
Leftover `traceShowId`

Reply:
Looks like a comment on an old diff? traceShowId doesn't appear in latest version.

Reply:
Oops, I tried to make sure I was looking at the latest but I guess I was not successful.